src/main/java/io/fabric8/elasticsearch/plugin/acl/BaseRolesSyncStrategy.java

```diff
@@ -49,10 +50,20 @@ protected String formatKibanaIndexName(OpenshiftRequestContext context, String k
     }
     
     
-    protected String formatKibanaRoleName(OpenshiftRequestContext context) {
-        if (context.isOperationsUser()) {
-            return SearchGuardRolesMapping.KIBANA_SHARED_ROLE;
-        } else {
+    public static String formatKibanaRoleName(OpenshiftRequestContext context) {
+        final String kibanaIndexMode = context.getKibanaIndexMode();
+        if (context.isOperationsUser() ) {
+            switch (kibanaIndexMode) {
+            case KibanaIndexMode.UNIQUE:
```

Comment:
can we indent our `case`s? IMO it makes switch statements easier to read...

Reply:
We cannot based on the current style rule.  It will have to be changed if that's what we want.  I, unfortunately, don't know what the quick win is to make that happen